addon/components/base/bs-form/element/control.js

```diff
@@ -16,6 +16,13 @@ export default Ember.Component.extend({
   value: null,
 
   /**
+   * @property ariaDescribedby
+   * @type {string}
+   * @public
+   */
+  ariaDescribedby: null,
```

Comment:
Yeah, I had it like this before, but then realized that the final attribute is actually `aria-describedby`. If the camel case version would be `ariaDescribedBy`, then the dasherized attribute name should be (at least to be consistent with Ember's inflection rules) `aria-described-by`, right?